test/unit/address-resolution-tests.js

```diff
@@ -7,7 +7,7 @@ var addressResolution = require('../../lib/policies/address-resolution');
 var EC2MultiRegionTranslator = addressResolution.EC2MultiRegionTranslator;
 
 describe('EC2MultiRegionTranslator', function () {
-  this.timeout(5000);
+  this.timeout(10000);
```

Comment:
I think this is the second time we've bumped this up, is weird how it can take > 5 secs at times, lets see if this fixes it :)